homeassistant/components/tasmota/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Tasmota (beta)",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/tasmota",
-  "requirements": ["hatasmota==0.1.4"],
+  "requirements": ["hatasmota==0.1.5"],
```

Comment:
The [lastest release](https://pypi.org/project/HATasmota/) on PyPI is 0.2.3. It looks like that the bump from 0.1.4 to 0.1.5 contains the same change as from 0.2.2 to 0.2.3.

Reply:
Yeah, but 0.2.x introduces some other features too which we don't want yet.